src/modules/coreHttpApi/openapi.yml

```diff
@@ -431,6 +442,146 @@ paths:
         403:
           $ref: "#/components/responses/Forbidden"
 
+  /api/v2/Attributes/{predecessorId}/Succeed:
+    post:
+      operationId: succeedAttribute
+      description: Succeeds either a Relationship Attribute or a Repository Attribute.
+      tags:
+        - Attributes
+      parameters:
+        - name: predecessorId
+          in: path
+          required: true
+          schema:
+            $ref: "#/components/schemas/AttributeID"
+          description: ID of the predecessor attribute
+      requestBody:
+        content:
+          application/json:
+            schema:
+              additionalProperties: false
+              properties:
+                successorContent:
+                  type: object
+                  additionalProperties: false
+                  required:
+                    - value
+                  properties:
+                    value:
+                      $ref: "#/components/schemas/AttributeValue"
+                    tags:
+                      $ref: "#/components/schemas/IdentityAttributeContent_tags"
+                    validFrom:
+                      $ref: "#/components/schemas/AttributeContent_validFrom"
+                    validTo:
+                      $ref: "#/components/schemas/AttributeContent_validTo"
+
+      responses:
+        200:
+          description: |
+            Success. Succeeding a Relationship Attribute will return an
```

Comment:
> Succeeding a Relationship Attribute will return an

Use present tense. predecessor/successor is missing, why "additional" ID?

Reply:
The ID is only set when a relationship attribute is suceeded as this will directly be send to the peer via notification